src/test/java/api/support/builders/RequestPolicyBuilder.java

```diff
@@ -0,0 +1,53 @@
+package api.support.builders;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.RequestType;
+
+import io.vertx.core.json.JsonObject;
+
+public class RequestPolicyBuilder extends JsonBuilder implements Builder {
+  private final UUID id;
+  private final String name;
+  private final String description;
+  private final RequestType type;
+
+  public RequestPolicyBuilder() {
+    this(UUID.randomUUID(),
+      "Example Request Policy",
+      "An example request policy",
+      RequestType.HOLD
+    );
+  }
+
+  private RequestPolicyBuilder(
+    UUID id,
+    String name,
+    String description,
+    RequestType type) {
+
+    this.id = id;
+    this.name = name;
+    this.description = description;
+    this.type = type;
+  }
+
+  @Override
+  public JsonObject create() {
+    JsonObject request = new JsonObject();
+
+    if (id != null) {
+      put(request, "id", id.toString());
+    }
+
+    put(request, "name", this.name);
+    put(request, "description", this.description);
+
+    JsonObject requestType = new JsonObject();
+    put(requestType, "type", this.type.name());
+
+    put(request, "requestType", requestType);
```

Comment:
This produces JSON like
```
{
  "id" : "59cb65ab-31d3-4aec-b34d-be76e14880c1",
  "name" : "Example Request Policy",
  "description" : "An example request policy",
  "requestType" : {
    "type" : "HOLD"
  }
}
```

Which does not match the [JSON schema for request policies](https://github.com/folio-org/mod-circulation-storage/blob/ec92fb24464481f44ce19e55a10c002dab944db4/ramls/request-policy.json#L18), please can this be amended.